rust/processor/src/db/parquet/models/fungible_asset_models/parquet_v2_fungible_asset_balances.rs

```diff
@@ -0,0 +1,183 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable},
+    db::common::models::fungible_asset_models::raw_v2_fungible_asset_balances::{
+        CurrentFungibleAssetBalanceConvertible, CurrentUnifiedFungibleAssetBalanceConvertible,
+        FungibleAssetBalanceConvertible, RawCurrentFungibleAssetBalance,
+        RawCurrentUnifiedFungibleAssetBalance, RawFungibleAssetBalance,
+    },
+};
+use allocative_derive::Allocative;
+use field_count::FieldCount;
+use lazy_static::lazy_static;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+lazy_static! {
+    pub static ref DEFAULT_AMOUNT_VALUE: String = "0".to_string();
+}
+
+#[derive(
+    Allocative, Clone, Debug, Default, Deserialize, FieldCount, ParquetRecordWriter, Serialize,
+)]
+pub struct FungibleAssetBalance {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub storage_id: String,
+    pub owner_address: String,
+    pub asset_type: String,
+    pub is_primary: bool,
+    pub is_frozen: bool,
+    pub amount: String, // it is a string representation of the u128
+    #[allocative(skip)]
+    pub block_timestamp: chrono::NaiveDateTime,
+    pub token_standard: String,
+}
+
+impl NamedTable for FungibleAssetBalance {
+    const TABLE_NAME: &'static str = "fungible_asset_balances";
+}
+
+impl HasVersion for FungibleAssetBalance {
+    fn version(&self) -> i64 {
+        self.txn_version
+    }
```

Comment:
are these traits only used by the non-sdk version of parquet processors?

Reply:
yep, we can deprecate after migrate all